data_collection/gazette/spiders/ba_ibitiara.py

```diff
@@ -0,0 +1,7 @@
+from gazette.spiders.base.portal_da_transparencia import PortalDaTransparenciaBaseSpider
+
+
+class BaIbitiaraSpider(PortalDaTransparenciaBaseSpider):
+    TERRITORY_ID = "2913002"
+    name = "ba_ibitiara"
+    start_urls = ["http://ba.portaldatransparencia.com.br/prefeitura/ibitiara/"]
```

Comment:
The last edition dates from 20/Janeiro/2012. Could you take a look at the current gazettes' website?

Reply:
https://www.ibitiara.ba.gov.br/site/diariooficial